recipes/doctr/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.1" %}
+package:
+  name: doctr
+  version: {{ version }}
+
+source:
+  fn: doctr-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/doctr/doctr-{{ version }}.tar.gz
+  md5: b5eed31c06c33f338ab303ee6e3e83a8
+
+build:
+  number: 0
+  entry_points:
+    - doctr = doctr.__main__:main
+  skip: true  # [not py35]
```

Comment:
Maybe it would be better to do `# [py<35]`. 😉 At some point we will have Python 3.6 (probably end of the year). Guessing you guys want to support that out-of-the-box too.